societe-francaise-degyptologie.csl

```diff
@@ -0,0 +1,212 @@
+              <if match="any" disambiguate="true">
+                <text macro="disambiguate-citation"/>
+              </if>
+              <else>
+                <text term="ibid" form="short" font-style="italic"/>
+              </else>
+            </choose>
+          </else-if>
+          <else-if match="any" position="subsequent">
+            <choose>
+              <if match="any" disambiguate="true">
+                <text macro="disambiguate-citation"/>
+              </if>
+              <else>
+                <text value="op. cit." font-style="italic" text-decoration="none"/>
+              </else>
+            </choose>
+          </else-if>
+          <else>
+            <choose>
+              <if type="chapter" match="any">
+                <group delimiter=" ">
+                  <text term="in"/>
+                  <text macro="editor"/>
+                </group>
+              </if>
+            </choose>
+            <choose>
+              <if type="article article-journal article-magazine article-newspaper" match="any">
+                <text macro="title"/>
+                <group delimiter=" ">
+                  <text macro="container-title"/>
+                  <text macro="volume"/>
+                  <text macro="date" prefix="(" suffix=")"/>
+                </group>
+              </if>
+              <else-if match="any" variable="volume">
+                <choose>
+                  <if type="book" match="any">
+                    <text macro="title"/>
+                  </if>
+                  <else-if type="chapter" match="any">
+                    <text macro="container-title"/>
+                  </else-if>
+                </choose>
+                <group delimiter=" ">
+                  <text macro="volume"/>
+                  <choose>
+                    <if match="any" variable="collection-title">
+                      <group delimiter=" " prefix="(" suffix=")">
+                        <text macro="collection"/>
+                        <text macro="collection-number"/>
+                      </group>
+                    </if>
+                  </choose>
+                </group>
+                <text macro="date"/>
+              </else-if>
+              <else>
+                <group delimiter=" ">
+                  <choose>
+                    <if type="book" match="any">
+                      <text macro="title"/>
+                    </if>
+                    <else-if type="chapter" match="any">
+                      <text macro="container-title"/>
+                    </else-if>
+                  </choose>
+                  <group delimiter=" " prefix="(" suffix=")">
+                    <text macro="collection"/>
+                    <text macro="collection-number"/>
+                  </group>
+                </group>
+                <text macro="date"/>
+              </else>
+            </choose>
+          </else>
+        </choose>
+        <text macro="locator"/>
```

Comment:
Similarly, it looks like there is no delimiter between the rest of the citation and the locator. Is that correct?

Reply:
The comma used as a delimiter by the group started on line 117 is not enough?